### PR TITLE
Filter for round video messages

### LINKED ISSUE #1440: Add filter for video messages
Telescope/video messages are now implemented (only playback) but currently not included in any filter. Create a filter for video messages, just like the one for voice messages.

### PR DESCRIPTION
**Pull request by stek29**
Fixes #1440

### CODE REVIEW

**Comment by stek29** [comment]
@zhukov I guess your comments are lost now because I amended and force pushed

File: app/js/messages_manager.js [hunk 1]
```diff
@@ -803,6 +803,11 @@ angular.module('myApp.services')
               neededDocType = 'voice'
               break
 
+            case 'inputMessagesFilterRoundVideo':
+              neededContents['messageMediaDocument'] = true
+              neededDocType = 'video'
```
Comment (zhukov) [inline app/js/messages_manager.js]: again, `neededDocType = 'round'` should be here.

**Comment by stek29** [comment]
@zhukov Last time I tried `round` caused "no messages found", and when I looked into document type of round videos it was "video". Maybe something changed or I made a mistake at first, idk. 
Sorry for long wait, had no chance to fix this since I wasn't able to use the laptop.

**Comment by zhukov** [comment]
Thanks for the PR

**Pull request merged**